Adyen.EcommLibrary/Model/ApplicationInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+namespace Adyen.EcommLibrary.Model
+{
+    [DataContract]
+    public class ApplicationInfo
```

Comment:
did you autogenerate this ? we need all of the options into the model.

Reply:
No I manually add it . Do we auto generate this models also from swagger?

Reply:
Now the models are generated from swagger